src/openzaak/config/setup_configuration/steps/selectielijst.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.models import Service
+
+from openzaak.config.setup_configuration.models import SelectielijstAPIConfig
+from openzaak.selectielijst.models import ReferentieLijstConfig
+
+
+class SelectielijstAPIConfigurationStep(BaseConfigurationStep[SelectielijstAPIConfig]):
+    """
+    Configure the Open Zaak client to request Selectielijst API
+
+    1. Create service for Selectielijst API
+    2. Set up configuration for Selectielijst API client
+
+    Normal mode doesn't change the allowed and default years
+    If they are changed, run this command with 'overwrite' flag
+    """
+
+    verbose_name = "Selectielijst API Configuration"
+    enable_setting = "openzaak_selectielijst_config_enable"
+    namespace = "openzaak_selectielijst_config"
+    config_model = SelectielijstAPIConfig
+
+    def execute(self, model: SelectielijstAPIConfig) -> None:
+
+        # 1. Set up a service for the Selectielijst API so Open Zaak can request it
+        service, created = Service.objects.update_or_create(
+            api_root=model.api_root,
+            defaults={
+                "label": "Selectielijst API",
+                "slug": model.api_root,
+                "api_type": APITypes.orc,
+                "oas": model.oas,
+                "auth_type": AuthTypes.no_auth,
+            },
+        )
```

Comment:
zgw-consumers has a step for this (https://zgw-consumers.readthedocs.io/en/latest/setup_config.html), so I think it would be better to reuse that step, on the `SelectielijstAPIConfig` model add a reference to the identifier https://github.com/open-zaak/open-notificaties/blob/main/src/nrc/setup_configuration/models.py#L10 and do something like this in the step to get the service: https://github.com/open-zaak/open-notificaties/blob/main/src/nrc/setup_configuration/authorization.py#L9

That way we don't duplicate the code for creating `Services` and if any changes are made to the `Service` model, it can be tackled in the library's step right away and no code in the projects has to be changed